test/unit/middleware/prepareQuery.js

```diff
@@ -206,6 +206,7 @@ describe('prepareQuery', () => {
       populate: [
         {
           path: 'foo',
+          strictPopulate: false,
```

Comment:
Correct me if I'm wrong but I think we shouldn't have to specify `strictPopulate: false` in these tests because we should default to the current behavior?

Reply:
Well, since the new default in Mongoose is to return an error, the only way (I've spotted so far) to maintain the current behavior in express-restify-mongoose is to apply `strictPopulate: false` on each populate path.

So, in my implementation, `strictPopulate: false` is automatically added to all populate queries (unless the user already specified a value).

Reply:
My bad, I misread what the test is doing. Looks good 👌